app/js/filters.js

```diff
@@ -16,7 +17,7 @@ angular.module('myApp.filters', ['myApp.i18n'])
       if (!user || !user.first_name && !user.last_name) {
         return _('user_name_deleted')
       }
-      return user.first_name + (user.last_name ? ' ' + user.last_name : '')
+      return user.first_name ? (user.first_name + (user.last_name ? ' ' + user.last_name : '')) : user.last_name
```

Comment:
I added this because of the if statement above, ``!user.first_name && !user.last_name``. 
@stek29 should I undo this "if" and remove the ``&& !user.last_name`` part? What do you think is the best way? or just leave it there and take the uncovered branch for granted?